build_sgx_binary.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Scripts should go into bin/ and not have the .sh extension.

However should this really be done in a build.rs file instead, so it always executed prior to the poet build?

Reply:
Will add a new commit to handle this